src/cplscheme/BaseCouplingScheme.cpp

```diff
@@ -657,34 +603,46 @@ bool BaseCouplingScheme::anyDataRequiresInitialization(BaseCouplingScheme::DataM
   return false;
 }
 
-bool BaseCouplingScheme::accelerate()
+bool BaseCouplingScheme::doImplicitStep()
 {
+  updateThisWindow();
```

Comment:
I am still a bit confused by this name. Wouldn't "updateWaveforms" be better?
Or "storeDataInWaveforms"?

Reply:
Fixed in 880edd9dd447723d8bcc9e9881435debfba7399a.